openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk

```diff
@@ -24,7 +24,7 @@ export VREG_AREA                 = 40 40 300 60
 export PDN_TCL 			= ../blocks/$(PLATFORM)/pdn.tcl
 
 #Placement options
-export PLACE_DENSITY = 0.40
+export PLACE_DENSITY = 0.4
```

Comment:
Have you tried pushing the place density further?

Reply:
Yes. At the moment the PLACE_DENSITY is changed as per number of switches. This was pushed in my previous PR

Reply:
Do we have a relationship between the number of switches and Load current?